opencti-platform/opencti-graphql/config/schema/opencti.graphql

```diff
@@ -5596,6 +5597,7 @@ input MalwareAddInput {
   objectLabel: [String]
   externalReferences: [String]
   killChainPhases: [String]
+  samples: [String]
```

Comment:
there should be changes in the generated file graphql.ts (when I started the backend I saw these changes locally)